sorting/select_k/select_k.py

```diff
@@ -0,0 +1,74 @@
+def select_k(arr, k):
+    """
+    Function of selecting k-th element of array
+    Parameters
+    ----------
+        arr:    list, input array
+        k:      int, index number in sorted array
+        
+    Examples
+    --------
+        >>> arr = [1, 9, 2, 8, 3, 7, 4, 6, 5]
+        >>> select_k(arr, 1)
+        2
+        >>> select_k(arr, -1)
+        9
+        >>> select_k(arr, 5) == sorted(arr)[5]
+        True
+    """
+    k = len(arr) + k if k < 0 else k
```

Comment:
where negative k values are used?

Reply:
It need if you want to select last element. In sorted array you just use `-1` index instead `len(arr) - 1`.